ios/app/brave_core_main.mm

```diff
@@ -93,6 +94,9 @@ - (instancetype)initWithUserAgent:(NSString*)userAgent
 
     ios::GetChromeBrowserProvider().Initialize();
 
+    web::WebUIIOSControllerFactory::RegisterFactory(
+        ChromeWebUIIOSControllerFactory::GetInstance());
```

Comment:
`ChromeWebUIIOSControllerFactory::GetInstance()` should be added to https://github.com/brave/brave-core/blob/master/chromium_src/ios/chrome/browser/browser_state/browser_state_keyed_service_factories.mm